WooCommerce/Classes/ViewRelated/Authentication/AccountCreationForm.swift

```
@@ -123,13 +123,13 @@ struct AccountCreationForm: View {
                 Button(Localization.submitButtonTitle) {
                     Task { @MainActor in
                         isPerformingTask = true
-                        let result = await viewModel.createAccount()
-                        isPerformingTask = false
-                        switch result {
-                        case .success:
+
+                        do {
+                            try await viewModel.createAccount()
```

Perhaps we can use `try?` here to simplify this block if we want to ignore the error case

---

Sure, done in 5b2efe2b3cda20c88dc945a3ab2ff2fb27740f3f